test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
@januszm `RGeo::Cartesian.preferred_factory` returns a `RGeo::Cartesian::Factory` for me, which includes SRID in equality. This is because `RGeo::Geos.supported?` returns false for me. `RGeo::Geos::FFI_SUPPORTED` and `RGeo::Geos::CAPI_SUPPORTED` are both false for me; which is true for you, and why? Do you have GEOS installed? (As I said above I didn't, and installing it didn't change this behavior. Not sure why not, actually, but let's see what your situation is first.)